package.json

```diff
@@ -11,12 +11,26 @@
     "index.js",
     "index.css",
     "themes.json",
+    "theme-data.js",
+    "theme-data.d.ts",
     "toucan.css",
     "plugins/**/*.js"
   ],
+  "exports": {
+    ".": "./index.js",
+    "./themes": "./themes.json",
+    "./themes.json": "./themes.json",
```

Comment:
Why the two entries for this file?

Reply:
Depending on import/require style, sometimes the .json extension is specified in the import path